cmd/xl-storage-disk-id-check.go

```diff
@@ -466,11 +466,15 @@ func (p *xlStorageDiskIDCheck) DeleteVersions(ctx context.Context, volume string
 	}
 	defer done(&err)
 
-	errs = p.storage.DeleteVersions(ctx, volume, versions)
+	w := xioutil.NewDeadlineWorker(diskMaxTimeout)
+	rerr := w.Run(func() error {
+		errs = p.storage.DeleteVersions(ctx, volume, versions)
```

Comment:
This is a batch call, so I think this should be applied at the lower level for each operation.

Since we know the batch size a lesser optimal version would be to multiply by the number of versions. Wouldn't recommend.